Fix indigo rendering in 19w42a

grondag opened the pull request:
Mojang moved matrix translation setup for chunk position up the stack into the methods that call `tesselateBlock.`  Indigo still applies the matrix to vertex data but the translation setup logic was removed.

grondag committed:
Fix indigo rendering in 19w42a

grondag commented:
This fixes terrain rendering with vanilla blocks, but seeing some remaining issues with modded models.  Can merge this now if you like and will do rest as a follow-on, or can wait.

modmuss50 commented:
Thanks for this. @sfPlayer1 can you take a peak at this should be good to merge.

grondag committed:
Prevent crashes for modded item rendering

VertexConsumer now thows if all vertex elements aren't populated.  Overlay UV coordinates are thus required for block breaking, item glint, etc.  Block breaking doesn't seem to work fully for non-vanilla models, but cause isn't yet clear.

Player3324 commented:
Looks good, thanks.
